src/realm/group.cpp

```diff
@@ -790,7 +790,9 @@ void Group::write(File& file, const char* encryption_key, uint_fast64_t version_
     REALM_ASSERT(file.get_size() == 0);
 
     file.set_encryption_key(encryption_key);
-    File::Streambuf streambuf(&file);
+    // Use big buffer to curb the cust of encryption
+    std::size_t buffer_size = 1048576; // 1 MiB
```

Comment:
consider using constexpr?

Reply:
I don't think there is a reason to use `constexpr` here. What effect are you looking for?